makejdk.sh

```diff
@@ -199,7 +210,11 @@ if [ "${USE_DOCKER}" ] ; then
     docker ps -a | awk '{ print $1,$2 }' | grep $CONTAINER | awk '{print $1 }' | xargs -I {} docker rm {}
   fi
 
-else
-  echo "Calling sbin/build.sh $WORKING_DIR $TARGET_DIR"
-  ./sbin/build.sh $WORKING_DIR $TARGET_DIR
-fi
+else  
+  echo "Calling sbin/build.sh $WORKING_DIR $TARGET_DIR $BUILD_FULL_NAME"
```

Comment:
Yeah best such changes or more in next PRs, keeps this PR focused to minimum changes to be merged